### PR TITLE
ui: added e2e tests for claiming - part 2

### PR DESCRIPTION
**Pull request by karolina-siemieniuk-morawska**

### CODE REVIEW

**Comment by drjova** [comment]
@karolina-siemieniuk-morawska aren't the screenshots missing 😕

**Comment by karolina-siemieniuk-morawska** [comment]
> @karolina-siemieniuk-morawska aren't the screenshots missing 😕

No, I didn't add any visual tests as claiming does not have any visual impact on the page, test results are based on the messages content that should match once claiming succeed/fails :)

File: e2e/cypress/integration/claiming.test.js [hunk 1]
```diff
@@ -0,0 +1,525 @@
...
+      it('assigns paper to user profile successfully', () => {
+        cy.intercept('POST', '/api/assign/literature/assign', {
+          statusCode: 200,
+          body: {
+            message: 'Success',
+          },
+        }).as('getAssignSuccess');
+
+        cy.visit('/authors/1010819');
+        cy.waitForLoading(50000);
```
Comment (drjova) [inline e2e/cypress/integration/claiming.test.js:32]: do we need to wait that much?

**Review by drjova — APPROVED** [review]
Just the comment about timeout, otherwise LGTM 🚀

**Pull request merged**